lib/.gitignore

```diff
@@ -27,3 +27,4 @@ nbdist/
 
 ### DEV-TOOLS ###
 .checkstyle
+checkstyleCacheFile
```

Comment:
see below. Should not be necessary if file is moved to targert

Reply:
I removed it